EIPS/eip-7528.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 7528
```

Comment:
```suggestion
eip: 7505
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename

Reply:
Fixed, thanks for the info